src/Database/Schema/Collection.php

```diff
@@ -25,6 +25,11 @@
  *
  * Used to access information about the tables,
  * and other data in a database.
+ *
+ * @method array<string> listTablesAndViews() Get the list of tables available in the current connection.
```

Comment:
Why does this need to be annotated here? The function is defined below in the class.

Reply:
I can clean this up in a follow up pull request.